src/expandable-section/interfaces.ts

```diff
@@ -47,10 +49,41 @@ export interface ExpandableSectionProps extends BaseComponentProps {
   /**
    * Heading displayed above the content text.
    *
-   * When using the container variant, use the Header component. Otherwise, use plain text.
+   * When using the container variant, use additional header props instead. Otherwise, use plain text.
    */
   header?: React.ReactNode;
 
+  /**
+   * The heading text. Use with container variant. Plain text is recommended.
+   */
+  headerText?: React.ReactNode;
```

Comment:
can we make it a string?

Reply:
Note: as discussed in the team sync, we decide to keep it as ReactNode.